content/post/stats_at_work.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "A chat about A/B testing"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
+tags: ["statistics", "ab_testing"]
+date: 2018-07-26T20:35:46-04:00
+draft: true
+---
+
+The following dialogue is taking place between a data scientist (DS) and a marketing analyst (MA).
+
+DS: The results are in...A is the winner!  
+MA: That's great - we all had a hunch that A would be better, but glad we tested it to be 100% sure.
+
+DS: Great. See you next time.  
+MA: Hang on a second. While I have you I had a quick question. This might sound dumb, but how do you know that A is actually better than B? I mean, after a few days I saw that A's average conversion rate was a bit higher than B's. I could've told you that A was better. Why did we have to run a test?
+
+<!--more-->
+
+DS: Right. But how do you know that B wouldn't have caught up to A eventually?  
+MA: Sure but what are the chances of that?
+
+DS: Yeah good question. We run tests to be able to answer questions like that - "What are the chances that even though A is currently better than B, that this trend continues"?  
+MA: I see. And how exactly do we answer that?
+
+DS: Well, we usually measure how confident we are that A is better than B using something called the p-va-  
+MA: Yeah p-value. I know about that. If the p-value is less than 5% then the test is significant otherwise it's not.
+
+DS: Yeah more or less.  
+MA: OK. And I know that the p-value is not the probability of A being better than B. It's something else...but what is it exactly?
+
+DS: The exact definition of the p-value is not super helpful, but it basically quantifies the probability of seeing what we saw - namely that A is better than B - if in fact A was the same as B! In other words, if A and B were equal, then how unlikely was our result?   
+MA: So a small p-value means that if A and B were equal, then our result would be highly unlikely. And this leads us to conclude that the result of the test can be trusted? It's accurate?
+
+DS: Kinda yeah. To be clear, there are different ways to quantify this kind of stuff. This type of analysis, called hypothesis testing, is the most common one used by the vast majority of scientists.   
+MA: Okay. So at the end of the test, you calculate the p-value and saw that it was under 5%. 
+
+DS: Right.  
+MA: A's overall conversion rate was 21% and B's 20.1%. So we're confident that A is bigger than B, cause we got a p-value less than 5%. But what if A's conversion rate was 20.2% instead? Or 25%? Don't we care about how much bigger A is?
+
+DS: Absolutely. There's some difference between A and B that we care about - say it's 0.5%. This is called the effect size. If the difference between A and B is less than 0.5%, then we say they're effectively equal. The p-value calculation takes this into account.   
```

Comment:
"Say it's 0.5%" here is a little confusing. Is it actually 0.5%? How would one know / choose the effect size? And **does** the p-value calculation take this into account?

Reply:
Yeah, I need to expand a bit on effect size. And tease out its relationship to p-value.